Tailscale/shared/Tailscale.sh

```diff
@@ -12,24 +15,31 @@ case "$1" in
         echo "${QPKG_NAME} is disabled."
         exit 1
     fi
-    mkdir -p -m 0755 /tmp/tailscale
-    if [ -e /tmp/tailscale/tailscaled.pid ]; then
-        PID=$(cat /tmp/tailscale/tailscaled.pid)
+
+    if [ -e ${PIDF} ]; then
+        PID=$(cat ${PIDF})
         if [ -d /proc/${PID}/ ]; then
           echo "${QPKG_NAME} is already running."
           exit 0
         fi
     fi
-    ${QPKG_ROOT}/tailscaled --port 41641 --statedir=${QPKG_ROOT}/state --socket=/tmp/tailscale/tailscaled.sock 2> /dev/null &
-    echo $! > /tmp/tailscale/tailscaled.pid
+    /bin/ln -sf ${QPKG_ROOT}/tailscaled /usr/bin/tailscaled
```

Comment:
I'd recommend doing this in the `package_routines` file in the `pkg_post_install` function, so it happens at install time and the links will be present even if Tailscale hasn't ever run.

To be consistent with other Linux platforms `tailscaled` would go in `/usr/sbin`